news/scaleto-max.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* new feature in `scale_to()` to run without specifying an x-value
```

Comment:
describe what it does, not what it doesn't do...i.e., scales to max value in each chase.  You can also add that it allows this to be used without xvalue